src/main/java/com/minecolonies/coremod/blocks/BlockHutBarracksTower.java

```diff
@@ -0,0 +1,48 @@
+package com.minecolonies.coremod.blocks;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Block of the BarracksTower.
+ */
+public class BlockHutBarracksTower extends AbstractBlockHut<BlockHutBarracksTower>
+{
+    /**
+     * Default constructor.
+     */
+    protected BlockHutBarracksTower()
+    {
+        //No different from Abstract parent
+        super();
+    }
+
+    @NotNull
+    @Override
+    public String getName()
+    {
+        return "blockHutBarracksTower";
+    }
+
+    /**
+     * This returns a complete list of items dropped from this block.
+     * @deprecated but we still need it.
+     * @param world   The current world
+     * @param pos     Block position in world
+     * @param state   Current state
+     * @param fortune Breakers fortune level
+     * @return A ArrayList containing all items this block drops
+     */
+    @Override
+    @Deprecated
+    public List<ItemStack> getDrops(final IBlockAccess world, final BlockPos pos, final IBlockState state, final int fortune)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)